_posts/2016-7-3-Bonsai-Water-1.md

```diff
@@ -0,0 +1,396 @@
+		</td>
+
+		<td style="text-align: center;">
+  		<a href="https://www.amazon.com/gp/product/B007XHZ25G/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B007XHZ25G&linkCode=as2&tag=predec-20&linkId=5a6e7dde482ca6cc0cf7ca66d0e937f0" target="\_blank"><img border="0" src="//ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&MarketPlace=US&ASIN=B007XHZ25G&ServiceVersion=20070822&ID=AsinImage&WS=1&Format=_SL110_&tag=predec-20" ></a><img src="//ir-na.amazon-adsystem.com/e/ir?t=predec-20&l=am2&o=1&a=B007XHZ25G" width="1" height="1" border="0" alt="DC 12V Submersible Mini Pump" style="border:none !important; margin:0px !important;" />
+		</td>
+
+		<td>
+			The specific brand of motor doesn't matter so much, but make sure it's fully submersible in water. We used a <a  href="https://www.amazon.com/gp/product/B00FIL6BMW/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B00FIL6BMW&linkCode=as2&tag=predec-20&linkId=35b3f2234feda1901d2078ccb2d95b66" target="\_blank">different motor</a><img src="//ir-na.amazon-adsystem.com/e/ir?t=predec-20&l=am2&o=1&a=B00FIL6BMW" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" /> at first, but that one died after about a month of use, so trying this one out now.
+		</td>
+	</tr>
+
+  <tr>
+		<td>
+      <a  href="https://www.amazon.com/gp/product/B008BHAOQO/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B008BHAOQO&linkCode=as2&tag=predec-20&linkId=3258d249474f1d2a8f66a582e5ffd168" target="\_blank">LM2596 DC-DC Buck Converter Step Down Module</a><img src="//ir-na.amazon-adsystem.com/e/ir?t=predec-20&l=am2&o=1&a=B008BHAOQO" width="1" height="1" border="0" alt="" style="border:none !important; margin:0px !important;" />
+		</td>
+
+		<td style="text-align: center;">
+      <a href="https://www.amazon.com/gp/product/B008BHAOQO/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B008BHAOQO&linkCode=as2&tag=predec-20&linkId=075fa7322bde177ea90210c0b5166d01" target="\_blank"><img border="0" src="//ws-na.amazon-adsystem.com/widgets/q?_encoding=UTF8&MarketPlace=US&ASIN=B008BHAOQO&ServiceVersion=20070822&ID=AsinImage&WS=1&Format=_SL110_&tag=predec-20" ></a><img src="//ir-na.amazon-adsystem.com/e/ir?t=predec-20&l=am2&o=1&a=B008BHAOQO" width="1" height="1" border="0" alt="LM2596 DC-DC Buck Converter Step Down Module" style="border:none !important; margin:0px !important;" />
+		</td>
+
+		<td>
+      This project is a bit more complicated than the last because it requires us to use 3 different magnitudes of voltage. The motor requires 12 V, but the Arduino can't run on anything more than about 5-7 V. Our sensor will require 3 V. This little module allows us to take the 12V voltage from the outlet and step it down to 5V to power the Arduino.
```

Comment:
"Magnitudes" of voltage doesn't sounds right. I think it should just be "three different voltages".